statchatui/src/Home/Home.jsx

```diff
@@ -1,16 +1,32 @@
-import React from "react";
+import React, { useState } from "react";
 import Stats from "../Stats/Stats";
 const GAME_NAMES = {
-  APEX: "Apex",
+  APEX: "Apex Legends",
+  R6: "Rainbow Six Siege",
 };
 const Home = ({ onStatsChange, stats }) => {
+  //FIXME initial state should store user's info from the database if it already exists
+  const [formData, setFormData] = useState({
+    gamertag: "",
+    platform: "",
+    gameName: "",
+  });
   const handleSubmit = async (event) => {
     event.preventDefault();
 
+    const gamertag = event.target.gamertag.value;
+    const platform = event.target.platform.value;
+    let gameName = event.target.gameName.value;
+
+    //i did this because the api requires the apex legends xbox field to be X1 instead of Xbox, but for r6, it needs to stay xbox
+    if (gameName === GAME_NAMES.APEX && platform === "Xbox") {
+      gameName = "X1";
+    }
+
     const formData = {
-      gamertag: event.target.gamertag.value,
-      platform: event.target.platform.value,
-      gameName: GAME_NAMES.APEX,
+      gamertag,
+      platform,
+      gameName,
```

Comment:
Nice! I like that you used the [shorthand notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions) here.